### ISSUE TITLE
Absolute time is annoying, how about relative time?

### ISSUE THREAD

**Issue by dylanh724** [hubot-archive/hubot-ambush#5]
"2 hours ago" instead of an entire sentence long for the time?

**Comment by jjasghar** [comment]
Makes sense. I wanted to do this with the [lita-ambush](https://github.com/jjasghar/lita-ambush) https://github.com/jjasghar/lita-ambush/issues/6 also, but i havent made it happen. 

Honestly it's above my knowledge of ruby too, or hell coffeescript for this matter.
